.rubocop.yml

```diff
@@ -1,2 +1,7 @@
 AllCops:
-  TargetRubyVersion: 2.5
\ No newline at end of file
+  TargetRubyVersion: 2.5
+
+Metrics/BlockLength:
+  Exclude:
+    - 'Gemfile'
+    - 'spec/**/*.rb'
```

Comment:
Do we want to add in the rules we use in our other projects here?

Reply:
Possibly. This is just what I used to stop my (bloated Java) IDE from complaining.

Reply:
Added plus autocorrected. 137 => 15 issues.